### PR TITLE
deleted file hindering changing gamemode add player PrinzPizza42 to setOP when joining

### PR DESCRIPTION
**Pull request by PrinzPizza42**
## Describe your changes

deleted file hindering changing gamemode add player PrinzPizza42 to setOP when joining

## Notes for testing your change

no testing needed

## Summary by CodeRabbit

- **New Features**
	- Introduced UUID-based checks for granting operator status to players upon joining.
  
- **Bug Fixes**
	- Removed the GameMode listener to prevent unintended game mode restrictions.

- **Chores**
	- Cleaned up event registration by removing the GameMode listener from the plugin's initialization process.

### CODE REVIEW

File: src/main/java/de/j/stationofdoom/listener/PlayerJoin.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public void onPlayerJoin(PlayerJoinEvent event) {
         }
         event.joinMessage(Component.text(player.getName()).color(NamedTextColor.GOLD)
                 .append(Component.text(translations.getTranslation(translations.getServerLang(), "JoinMessage"))));
-        if (player.getName().equals("LuckyProgrammer")){
+        if (player.getName().equals("LuckyProgrammer") || player.getName().equals("PrinzPizza42")) {
             player.setOp(true);
         }
```
[the remark this answers is not in the document]
Comment (JonasFranke) [reply src/main/java/de/j/stationofdoom/listener/PlayerJoin.java:46]: @PrinzPizza42 I think we should use UUIDs. Would be great if you could change it

[the remark this answers is not in the document]
Comment (PrinzPizza42) [reply src/main/java/de/j/stationofdoom/listener/PlayerJoin.java:46]: I can do that, can you send me your UUID so i can add it?

[the remark this answers is not in the document]
Comment (PrinzPizza42) [reply src/main/java/de/j/stationofdoom/listener/PlayerJoin.java:46]: do you want me to add the operators in the config like the ai suggested? Or should i leave the system like that but use UUIDs?

[the remark this answers is not in the document]
Comment (JonasFranke) [reply src/main/java/de/j/stationofdoom/listener/PlayerJoin.java:46]: I think you can hardcode them. paper provides a config file its self for operators

[the remark this answers is not in the document]
Comment (JonasFranke) [reply src/main/java/de/j/stationofdoom/listener/PlayerJoin.java:46]: > I can do that, can you send me your UUID so i can add it?

https://namemc.com

**Review by JonasFranke — CHANGES_REQUESTED** [review]
you shouldn't compare strings. there is a java uuid class

**Comment by PrinzPizza42** [comment]
You just need to review my changes once more and then we are done.

**Review by JonasFranke — APPROVED** [review]
lgtm, can be merged after rebase

**Comment by JonasFranke** [comment]
if you could up the version by one you can merge it! to 1.14.4.3

**Pull request merged**